cmd/ak/cmd/deployments/delete.go

```diff
@@ -0,0 +1,40 @@
+package deployments
+
+import (
+	"github.com/spf13/cobra"
+
+	"go.autokitteh.dev/autokitteh/cmd/ak/common"
+	"go.autokitteh.dev/autokitteh/internal/resolver"
+)
+
+var deleteCmd = common.StandardCommand(&cobra.Command{
+	Use:     "delete <deployment ID> [--fail]",
+	Short:   "Delete inactive deployment",
```

Comment:
technically the delete code above drains a deployment, so it deletes active deployments as well

Reply:
explain?

Reply:
you say delete inactive deployments in the text
but delete allows deleting active deployment, it just moves it to inactive first by draining it

Reply:
you are right. I already asked this question Haim and Itay. Got no answer meanwhile

Reply:
rewritten. Now we are checking that deployment is inactive and then deleting, err otherwise